catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/NsiliAttributeMap.java

```diff
@@ -203,16 +205,14 @@
   public static String getDdfAttributeForNsili(String nsiliAttribute, boolean removeSourceLibrary) {
     String attribute = nsiliAttribute;
     if (attribute.contains(".")) {
-      attribute = attribute.substring(attribute.lastIndexOf(":") + 1);
+      attribute = attribute.substring(attribute.lastIndexOf(':') + 1);
     }
     String attributeName = nsiliToDdfMap.get(attribute);
 
-    if (removeSourceLibrary) {
-      if (nsiliAttribute != null
-          && nsiliAttribute.equalsIgnoreCase(
-              NsiliConstants.NSIL_CARD + "." + NsiliConstants.SOURCE_LIBRARY)) {
-        attributeName = null;
-      }
+    if (removeSourceLibrary
```

Comment:
❓ Why did we remove the `nsiliAttribute != null` check? Seems like we could be getting an NPE now

Reply:
Already effectively tested on line 207, as `attribute` == `nsiliAttribute`.